docs/corpora_schema.md

```diff
@@ -11,19 +11,20 @@ Date Last Modified: 2020-12-11
 
 ## Background
 
-cellxgene aims to support the publication and sharing of single-cell datasets as well as the construction of a data corpus that facilitates data integration
-across multiple tissues and experiments. Achieving the latter goal requires some harmonization of metadata and features in the cellxgene Data Portal. But if
-that harmonization is too onerous, it will burden the goal of rapid data sharing.
+cellxgene aims to support the publication, sharing, and exploration of single-cell datasets.
+Building on those published datasets, cellxgene seeks to create references of the phenotypes and composition of cells that make up human tissues.
+Creating references from multiple datasets requires some harmonization of metadata and features in the cellxgene Data Portal. But if that harmonization is too onerous, it
+will burden the goal of rapid data sharing.
 
-We balance these two goals by requiring datasets hosted in the cellxgene Data Portal to follow a small schema with only a few required fields. These fields
+We balance publishing and reference creation needs by requiring datasets hosted in the cellxgene Data Portal to follow a small schema with only a few required fields. These fields
 are expected to be very useful for data integration and also simply and readily available from data submitters.
 
-Note that the requirements in the schema are just the minimum required information. We expect that datasets will have additional metadata, and this
-will be preserved in datasets submitted to the Data Portal.
+Note that the requirements in the schema are just the minimum required information. Datasets often have additional metadata, and this
+is preserved in datasets submitted to the Data Portal.
```

Comment:
Datasets often have additional metadata, ~and this~ [which] is preserved in datasets submitted to the Data Portal.

I'm on the fence about whether collisions require further discussion in this document or simply in the curation tools.

Reply:
I've implemented the proposed edit. 

I didn't add any commentary about collisions at present. My weak opinion is that it belongs in the schema guide, since that document relates to application of a schema to a dataset.